config/pipeline.yaml

```
@@ -908,6 +908,162 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.iommu
 
+  ltp: &ltp
+    template: ltp.jinja2
+    kind: job
+    nfsroot: 'https://storage.kernelci.org/images/rootfs/debian/bookworm-ltp/20240313.0/{debarch}'
+    params: &ltp-params
+      skip_install: "true"
+      tst_cmdfiles: ""
+      job_timeout: '15'
+      skipfile: skipfile-lkft.yaml
+    kcidb_test_suite: ltp
+    rules:
+      - blocklist: &kselftest_defconfig_filter
+        defconfig: ['kselftest']
+
+  ltp-cap-bounds:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "cap_bounds"
+
+  ltp-cpuhotplug:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "cpuhotplug"
+
+  ltp-crypto:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "crypto"
+    rules:
+      - passlist:
+          defconfig: ['crypto']
+
+  ltp-dio:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "dio"
+
+  ltp-fcntl-locktests:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "fcntl-locktests"
+
+  ltp-filecaps:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "filecaps"
+
+  ltp-fsx:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "fsx"
+
+  ltp-hugetlb:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "hugetlb"
+
+  ltp-ima:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "ima"
+    rules:
+      - passlist:
+          defconfig: ['ima']
+
+  ltp-io:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "io"
+
+  ltp-ipc:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "ipc"
+
+  ltp-mm:
+    <<: *ltp
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "mm"
+      job_timeout: '30'
```

```suggestion
      job_timeout: 30
```

(same change needed for all other occurences of this param)